apps/api/src/services/data-provider/data-provider.service.ts

```diff
@@ -667,7 +667,9 @@ export class DataProviderService {
 
     for (const date in rootData) {
       data[date] = {
```

Comment:
Would it also work to make the check here (and skip that date)?

Reply:
You're right, looks even better.
Adapted it accordingly